.github/workflows/release.yml

```diff
@@ -5,7 +5,7 @@ on:
       - "*"
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Check what fail in checks

Reply:
It seemed like it attempted to run the master but the master would install latest werkzeug version and would fail importing the function that was added in this PR into `utils.py`.
![image](https://user-images.githubusercontent.com/32478989/186588107-807f3c81-b151-4bab-847d-aea3c0227364.png)


Not sure what to do about it, However, after this PR, this issue (in particular, should not happen anymore).